docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -727,6 +727,27 @@ Your new custom cloud is ready to use only after its status changes to
      --display-name "CUSTOM_CLOUD_DISPLAY_NAME"
    ```
 
+   Replace the following:
+
+   - `ORGANIZATION_IDENTIFIER` with the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud,
+     for example `org484338bebd2`. You can extract `ORGANIZATION_IDENTIFIER` from the
+     output of the `avn organization list` command.
+   - `DEPLOYMENT_MODEL_NAME` with the type of [network architecture](/docs/platform/concepts/byoc#byoc-deployment)
+     your custom cloud uses:
+     - `standard_public` (public) model allows the Aiven control plane to connect to the
```

Comment:
The crucial difference between the deployment models, and why they exists, isn't so much how Aiven control plane connects to the VMs, but from where it is possible for service users to connect to the services. In `standard` the nodes reside in a VPC without public IP addresses and are by default not accessible from outside, in `standard_public` the nodes have public IPs and can be made publicly accessible. This is the important distinction  between the models to emphasize I think.